### PR TITLE
mtl: add dts-codec support

### PR DESCRIPTION
**Pull request by macchian**
add DTS codec support on MTL

### CODE REVIEW

**Review by lgirdwood — DISMISSED** [review]

**Comment by lgirdwood** [comment]
@kv2019i pls review.

**Review by lgirdwood — DISMISSED** [review]

File: config/mtl.toml [hunk 1]
```diff
@@ -630,3 +630,22 @@ count = 25
 	pin = [0, 0, 0xfeef, 0xf, 0xf, 0x45ff, 1, 0, 0xfeef, 0xf, 0xf, 0x1ff]
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 1000000, 128, 128, 0, 0, 0]
+
+	# dts codec module config
+	[[module.entry]]
+	name = "DTS"
```
Comment (btian1) [inline config/mtl.toml:636]: is dts PORed for mtl chrome book? also, how to test DTS codec functionality?

**Review by kv2019i — CHANGES_REQUESTED** [review]
Please resubmit directly to SOF repo (rimage moved to sof/tools/rimage).

**Comment by marc-hb** [comment]
you can very easily move these commits to the main sof repo, see how at:
- https://github.com/thesofproject/sof/pull/8297#issuecomment-1758566134

File: config/mtl.toml (hunk 1 above)
Comment (macchian) [reply config/mtl.toml:636]: @btian1 , not POR, but customer demands. I am working with DTS engineer @joechengxperi , we are preparing to submit the relevant DTS topology conf as well. The DTS EQIIR function is working based on them. 
One question:
Currently we set the default value on mod_cfg. Do you or someone know how to tuning the mod_cfg values ? Is there a tool to do that?

	mod_cfg = [0, 0, 0, 0, 296, 5000000, 384, 384, 0, 5000, 0]

**Comment by macchian** [comment]
> Please resubmit directly to SOF repo (rimage moved to sof/tools/rimage).

@kv2019i , @marc-hb , ok, I will resubmit to /sof/tools/rimage .

File: config/mtl.toml (hunk 1 above)
Comment (btian1) [reply config/mtl.toml:636]: I don't know how to tunning this, maybe @kv2019i @singalsu  know this part.
one more question is: we already have component EQIIR, what's the difference between existing EQIIR and DTS EQIIR?

Comment (joechengxperi) [reply config/mtl.toml:636]: To clarify that, we use existing EQ IIR + DTS currently, not DTS EQIIR.

Comment (singalsu) [reply config/mtl.toml:636]: That's nice to hear, don't hesitate to suggest improvements for IIR component or library if you come up with ideas.

**Comment by macchian** [comment]
> > Please resubmit directly to SOF repo (rimage moved to sof/tools/rimage).
> 
> @kv2019i , @marc-hb , ok, I will resubmit to /sof/tools/rimage .

Close this. New open #8326

**Pull request closed without merging**